source/shared/_cookie_policy_script.html.erb

```diff
@@ -0,0 +1,31 @@
+<body onload="checkCookie()"></body>
+<script>
+  function setCookie(cname, cvalue) {
```

Comment:
Here's how GOVUK cookies are managed: https://github.com/alphagov/govuk_template/blob/master/source/assets/javascripts/cookie-functions.js

govuk_template doesn't seem to be available to the govwifi-product-page build [other than as a development dependency](https://github.com/alphagov/govwifi-product-page/blob/master/package.json#L29) but perhaps it would be a good basis for functions here.

Reply:
@steventux good find. We do use `govuk_template`, we include a bunch of things in the `manifest.css.scss` file. But as the `govuk_template` is now deprecated, we've been trying to not use it any more. We've looked at removing it completely but its a bit of work to migrate everything over to the new design system. 

So in this instance, even though its a bit of a snowflake recreating this for ourselves, I think its probably better than further reliance on a deprecated tool.